### DISCUSSION TITLE
Is the Heart On Your Sleeve Badges still achievable?

### DISCUSSION THREAD

**Discussion by vickybelario** [community/community#138401] [Discussions] [answered]
### Select Topic Area

Question

### Body

I was trying to get GitHub badges and I need the Heart On Your Sleeve, I alredy react more than 2 love emoji and still dont got it

**Comment by LiteBrite82** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

**Comment by dhans11** [comment, marked as answer]
this badges is no longer achievable
